index.html

```diff
@@ -310,23 +314,17 @@ <h3>The <dfn><a>PerformanceMark</a> Constructor</dfn></h3>
             Set <var>entry</var>'s <code>startTime</code> attribute as follows:
             <ol>
               <li>
-                If <var>markOptions</var> is present and its <a>startTime</a> member is present, then:
+                If <var>markOptions</var>'s <a>startTime</a> member is present, then:

```

Comment:
Why did you remove the check to see if `markOptions` is given?

Reply:
In https://heycam.github.io/webidl/#idl-operations it says:
"If the type of an argument is a dictionary type or a union type that has a dictionary as one of its flattened member types, and that dictionary type and its ancestors have no required members, and the argument is either the final argument or is followed only by optional arguments, then the argument must be specified as optional. Such arguments are always considered to have a default value of an empty dictionary, unless otherwise specified."
I interpret this as meaning that those arguments are considered to always be given, but not sure about this...

Reply:
@heycam do you know if the check for whether this argument is given or not should be done?

Reply:
Yep, it's always given, per your reading. I agree it's a little non-obvious. It would be more obvious to require everyone to put `= {}` after it. https://github.com/heycam/webidl/issues/602

Reply:
Cool, thanks!